src/go/rdctl/pkg/process/process_windows_test.go

```diff
@@ -0,0 +1,31 @@
+package process
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/sys/windows"
+)
+
+func TestBuildCommandLine(t *testing.T) {
+	t.Parallel()
+	cases := [][]string{
+		{"arg0", "a b c", "d", "e"},
+		{"C:\\Program Files\\arg0\\\\", "ab\"c", "\\", "d"},
```

Comment:
Should there also be a test for mixed forward and backward slashes or is that too crazy?
e.g `C:/Program Files\\Test`

Reply:
Added test. This also happened to catch an error when there are no arguments :)